homeassistant/components/glances/config_flow.py

```diff
@@ -7,27 +7,46 @@
 import voluptuous as vol
 
 from homeassistant import config_entries, exceptions
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import (
     CONF_HOST,
     CONF_PASSWORD,
     CONF_PORT,
+    CONF_SCAN_INTERVAL,
     CONF_SSL,
     CONF_USERNAME,
     CONF_VERIFY_SSL,
 )
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, callback
 from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaFlowFormStep,
+    SchemaOptionsFlowHandler,
+)
 
 from . import get_api
 from .const import (
     CONF_VERSION,
     DEFAULT_HOST,
     DEFAULT_PORT,
+    DEFAULT_SCAN_INTERVAL,
     DEFAULT_VERSION,
     DOMAIN,
     SUPPORTED_VERSIONS,
 )
 
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(
+            CONF_SCAN_INTERVAL, default=DEFAULT_SCAN_INTERVAL
```

Comment:
We don't allow scan interval as option. Every integration that supports config entries allows the user to turn off automatic polling and then the user can automate the update interval as needed with the entity service `homeassistant.update_entity`.

Reply:
Huh? I found exactly the same for quite a lot of other integrations...? Is this just old code?
Sad I only knew after putting in that much effort :(. Nevermind, it is as it is

Reply:
Yes that's legacy code. If you're unsure about something and it's not documented in our dev docs or architecture records it's best to ask in our discord server.